utils/bazel/llvm-project-overlay/mlir/BUILD.bazel

```diff
@@ -6187,6 +6187,9 @@ cc_library(
         ":NVVMToLLVMIRTranslation",
         ":TargetLLVM",
         ":ToLLVMIRTranslation",
+        "@cuda//:cuda_headers",
+        "@cuda//:cusparse_static",
+        "@cuda//:libcuda",
```

Comment:
Are you sure that these two are necessary? I ran this internally without those two lines and the few hand-picked tests I tried worked fine...

Reply:
They were needed for nvPTXCompilerCreate and the corresponding header file iirc